### PR TITLE
[stable23] Fix settings navigation order

### PR DESCRIPTION
**Pull request by skjnldsv**
Backport of https://github.com/nextcloud/firstrunwizard/pull/696

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@ The First run wizard can be customized to meet specific design goals, or to chan
 			<id>firstrunwizard-about</id>
 			<name>About</name>
 			<route></route>
-			<order>5</order>
+			<order>65</order>
```
Comment (juliushaertl) [inline appinfo/info.xml:28]: Any reason to not use just 6? 

https://github.com/nextcloud/firstrunwizard/pull/696/files

Comment (skjnldsv) [reply appinfo/info.xml:28]: A typo :see_no_evil:

**Review by juliushaertl — APPROVED** [review]

**Comment by juliushaertl** [comment]
Merging without DCO as it is a very minor change.

**Pull request merged**